sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ToolTips.scala

```diff
@@ -29,4 +29,16 @@ private[ui] object ToolTips {
 
   val THRIFT_SERVER_DURATION =
     "Difference between start time and close time"
+
+  val THRIFT_SESSION_TOTAL_EXECUTE =
+    "Number of operations submitted in this session."
+
+  val THRIFT_SESSION_START_TIME =
+    "Session Start time,on launching the session."
+
+  val THRIFT_SESSION_FINISH_TIME =
+    "Session end time, after closing the session."
+
+  val THRIFT_SESSION_DURATION =
+    "Active duration of the session."
```

Comment:
Please add tooltips for all columns. You can refer to our web-ui documentation:
https://github.com/apache/spark/blob/d334fee502fa85cfc6ef1b7f193fa3a0ff0b244e/docs/web-ui.md#L416-L447

Reply:
@wangyum  in the same page statistics table does not have tooltip for all the column. Should we address that too?

Reply:
@wangyum  i have added tooltip for all the columns in session table. Could you review?

Reply:
I don't think we need tooltips for every single column.  "IP of the session" does not add anything over the column name. We should only add tooltips where they add something meaningful, like, clarifying for duration fields what the start and end time are exactly.

Reply:
Duration tooltips can basically specify one useful thing: what is the start and end time? this isn't telling the user anything more right now